tests/acceptance/async_adapter_tests.py

```diff
@@ -2,9 +2,18 @@
 import uuid
 
 from pika import spec
+from pika import compat
 
 from async_test_base import (AsyncTestCase, BoundQueueTestCase, AsyncAdapters)
 
+
+def _p3_as_bytes(value):
```

Comment:
@everilae: Instead of copying this function from blocking_adapter_tests.py, let's have a single shared version instead. Perhaps it could be a function in the pika.compat module?